cmd/get.go

```diff
@@ -2,18 +2,42 @@ package cmd
 
 import (
 	"fmt"
+	"log"
+	"regexp"
 
 	"github.com/pbs/gorson/internal/gorson/io"
 	"github.com/pbs/gorson/internal/gorson/json"
 	"github.com/pbs/gorson/internal/gorson/util"
 	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
 )
 
+var format string
+
 func get(path string) {
```

Comment:
get can be a switch here too... any thoughts on one or the other?

```
func get(path string) {
        p := util.NewParameterStorePath(path)
        pms := io.ReadFromParameterStore(*p)
        switch format {
        case "yaml", "yml":
                serialized, err := yaml.Marshal(pms)
                if err != nil {
                        log.Fatal(err)
                }
                fmt.Println(string(serialized))
        case "env":
                serialized, err := yaml.Marshal(pms)
                if err != nil {
                        log.Fatal(err)
                }
                var re = regexp.MustCompile(`(\w*):\s`)
                fmt.Println(re.ReplaceAllString(string(serialized), "${1}="))
        case "json":
                marshalled := json.Marshal(pms)
                fmt.Println(marshalled)
        default:
                log.Fatal("No proper format requested. (yaml, env, json allowed)")
        }
}
```

Reply:
I think case/switch is more idiomatic in go, but this is fine. No strong opinion :)